src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,121 @@
+package duke.task;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+import duke.exception.DukeException;
+
+/**
+ * The tasklist represents the list of tasks.
+ */
+public class TaskList extends ArrayList<Task> {
+    private static final long serialVersionUID = 200;
```

Comment:
Since this is a constant, may want to name it all uppercase using underscore to separate words e.g. `SERIAL_UID`

Reply:
>`serialVersionUID` is a special variable found in the Serializable Interface that helps to determine the Class version. Thus, the name cannot change without interfering with the function of the Class.
As per above.